scripts/deploy.sh

```diff
@@ -133,6 +133,13 @@ openstack-ansible -i "localhost," patcher.yml
 
 # begin the openstack installation
 if [[ "${DEPLOY_OA}" == "yes" ]]; then
+
+  # This deploy script is also used for minor upgrades (within an openstack release)
+  # Some versions of liberty deploy pip lockdown to the repo server, in order for an
+  # upgrade to suceed the pip config must be removed so that repo builds
+  # have access to external repos.
+  ansible repo_all -m file -a 'name=/root/.pip state=absent'
```

Comment:
This causes the initial deployment to fail because the containers don't exist.
I think it would be worth putting a note saying when this should be removed. Do we still work on the assumption that support deploy all minors? If so this would only need to be in 12.2, otherwise it needs to be here till 13.1.